### PR TITLE
Removed attachments for urls

### PR DESCRIPTION
**Pull request by TheWyn**
It's much easier with url's when using it through another cog (Trusty's retrigger for one)

### CODE REVIEW

**Comment by Flame442** [comment]
You replaced tabs with spaces, if nothing else that needs to change since I cannot tell what the diff is...

**Comment by TheWyn** [comment]
Removed message attachment for image url and included embed thumbnail url option.

Couldn't attach an image when wrapping this through another cog, thought it'd be helpful.

File: simpleembed/simpleembed.py [hunk 1]
```diff
@@ -6,35 +6,37 @@
...
+	async def sendembed(self, ctx, color: Optional[discord.Color] = None, image=None, thumbnail=None, *, text):
 		"""
 		Send an embed.
-		
+
 		Use the optional parameter `color` to change the color of the embed.
+		Use the optional parameter 'image' to include an image url
+		Use the optional parameter 'thumbnail' to include an thumbnail image url
 		The embed will contain the text `text`.
-		All normal discord formatting will work inside the embed.
```
Comment (Flame442) [inline simpleembed/simpleembed.py:21]: This line should not be removed.

File: simpleembed/simpleembed.py [hunk 2]
```diff
@@ -6,35 +6,37 @@
...
+
 	def __init__(self, bot):
 		self.bot = bot
-	
+
 	@checks.has_permissions(manage_messages=True)
 	@commands.bot_has_permissions(embed_links=True)
 	@commands.command()
-	async def sendembed(self, ctx, color: Optional[discord.Color]=None, *, text):
+	async def sendembed(self, ctx, color: Optional[discord.Color] = None, image=None, thumbnail=None, *, text):
```
Comment (Flame442) [inline simpleembed/simpleembed.py:16]: This signature **requires** both an image and a thumbnail be provided.

File: simpleembed/simpleembed.py [hunk 3]
```diff
@@ -6,35 +6,37 @@
...
 			color=color
 		)
-		if ctx.message.attachments:
-			content = await ctx.message.attachments[0].to_file()
-			embed.set_image(url="attachment://" + str(content.filename))
-		await ctx.send(embed=embed, file=content if ctx.message.attachments else None)
-		try:
-			await ctx.message.delete()
-		except discord.Forbidden:
-			pass
```
Comment (Flame442) [inline simpleembed/simpleembed.py:37]: This should not be removed.

**Review by Flame442 — CHANGES_REQUESTED** [review]
This change does not function as expected, and I'm not sure how you would manage to make it function as you want to. Other behavior and stylistic choices modified in this PR should not be touched.

The point of this cog is to be a **very** simple embed cog, and I don't think it would be possible to support images via url in a non-complex way. If you can find one, I'm fine with considering it, but as it stands right now, this PR breaks functionality.

**Comment by TheWyn** [comment]
Added to message delete back but couldn't figure out how to make those parameters optional, I thought `Optional[str] = None` would've worked but that only brought up the help menu.

**Comment by Flame442** [comment]
Since all the args (besides the first one) are of type `str`, they will all "want" to be used. The only way to make it not required would be to have something like `typing.Optional[LinkConverter]`, however such a converter would be unnecessarily complex for this cog.

**Comment by TheWyn** [comment]
Ahk nevermind then.

**Pull request closed without merging**